zram-generator.conf.example

```diff
@@ -6,14 +6,11 @@
 #
 # "host-memory-limit = none" may be used to disable this limit. This
 # is also the default.
-#
-# To disable a device (when some configuration for it already exits),
-# set "host-memory-limit = 0".
```

Comment:
hmm, why do we remove this?

Reply:
Misleading: anything that will lead to disksize=0 will disable a device. This is just about the noisiest method for doing so, and max size/fraxion are better for this.